assets/charts/control-plane/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -34,24 +11,19 @@ metadata:
     k8s-app: kube-apiserver
 spec:
   {{- if eq (int .Values.apiserver.replicas) 1 }}
-  replicas: 1
+  # Always run two replicas when it is single controller node.
```

Comment:
Why? `kube-apiserver` scales very well and it does not require 2 instances on the node, as it will have bigger memory footprint.

Reply:
removed and when it is deployment it is one node with tolerance to 0 replicas down when upgrading.

Reply:
Sounds good.